.travis.yml

```diff
@@ -105,7 +105,6 @@ install:
   - moodle-plugin-ci install -vvv
 
 script:
-  - moodle-plugin-ci phplint
```

Comment:
We cannot drop support for PHP 5.6 as it is used in many clients. @dmitriim has a solution that works in both

Reply:
@ilya-catalyst if you'd like to support older versions of moodle, then you can use polyfil see related docs https://docs.moodle.org/dev/Privacy_API#Difference_between_Moodle_3.3_and_more_recent_versions